lib/src/_isolate.dart

```diff
@@ -4,111 +4,32 @@ typedef IsolateRunnable<Req, Res> = Res Function(Req req);
 typedef IsolateCallback<Res> = void Function(Res res);
 
 class _IsolateTasker<Req, Res> {
-
   final String name;
-  final ReceivePort _receivePort;
   late bool _closed;
 
-  Isolate? _isolate;
-  int _receivedMessageIndex;
-  int _sentMessageIndex;
-  SendPort? _sendPort;
-  late List<_IsolateTask<Req, Res>> _taskQueue;
+  late IsolateManager<Res, Req>? _isolateManager;
 
-  _IsolateTasker(this.name, IsolateRunnable<Req, Res> runnable) :
-    _receivePort = ReceivePort(name),
-    _receivedMessageIndex = -1,
-    _sentMessageIndex = 0 {
+  _IsolateTasker(this.name, IsolateRunnable<Req, Res> runnable) {
     _closed = false;
-    _receivePort.listen((message) {
-      _receivedMessageIndex++;
-      if (_receivedMessageIndex == 0) {
-        _sendPort = message;
-        _runTaskQueue();
-      } else {
-        final int taskIndex = _taskQueue.indexWhere((element) => element.id == message.id);
-        if (taskIndex < 0) {
-          return;
-        }
-        final _IsolateTask<Req, Res> task = _taskQueue.removeAt(taskIndex);
-        task.callback(message.response);
-      }
-    });
-    _taskQueue = [];
-    final _IsolateInitMessage<Req, Res> message = _IsolateInitMessage<Req, Res>(_receivePort.sendPort, runnable);
-    Isolate.spawn<_IsolateInitMessage<Req, Res>>(_run, message).then((value) {
-      _isolate = value;
-    });
+    _isolateManager = IsolateManager.create(
+      runnable,
```

Comment:
> You must add @pragma('vm:entry-point') annotation to all methods that you want to use for isolation since Flutter 3.3.0. Without this annotation, the Dart compiler could strip out unused functions, inline them, shrink names, etc., and the native code would fail to call it.

I saw this on the docs of [isolate_manager](https://pub.dev/packages/isolate_manager), should we add the annotation?

Reply:
I'm really sorry, this was my mistake. I've been using it normally before, which led me to overlook this detail. It does need to be included; I've already added the necessary annotations.